admin/class-config.php

```diff
@@ -59,11 +59,19 @@ function config_page_scripts() {
 		wp_enqueue_script( 'wpseo-admin-script', plugins_url( 'js/wp-seo-admin' . WPSEO_CSSJS_SUFFIX . '.js', WPSEO_FILE ), array(
 			'jquery',
 			'jquery-ui-core',
+			'jquery-ui-progressbar',
 		), WPSEO_VERSION, true );
 		wp_localize_script( 'wpseo-admin-script', 'wpseoAdminL10n', $this->localize_admin_script() );
 		wp_enqueue_script( 'dashboard' );
 		wp_enqueue_script( 'thickbox' );
 
+        wp_enqueue_script( 'js-seo-config.js', plugins_url( 'js-seo/js/config/config.js' ), null, '2.2.1', true );
```

Comment:
Having an array of handle => filepath, and looping through them to enqueue each, would be cleaner.

~~~php
$js_seo_scripts = array(
    'js-seo-config.js' => 'config/config.js',
    'js-seo-scoring.js' => 'config/scoring.js',
    ...
);
foreach( $js_seo_scripts as $handle => $file ) {
    wp_enqueue_script( $handle, plugins_url( 'js-seo/js/' . $file ), null, '2.2.1', true );
}
~~~
}

----

Better yet, merge these into a single minified script before release and just have a single file in use.